gateway/public/index.html

```diff
@@ -5,7 +5,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta
       name="description"
-      content="A unified gateway solution designed to seamlessly connect and access files or folders permanently stored on Autonomys Mainnet&Testnet through a single, streamlined interface."
+      content="A unified gateway solution designed to seamlessly connect and access files or folders permanently stored on Autonomys Mainnet  & Taurus Testnet through a single, streamlined interface."
```

Comment:
```suggestion
      content="A unified gateway solution designed to seamlessly connect and access files or folders permanently stored on Autonomys Mainnet & Taurus Testnet through a single, streamlined interface."
```
Just fixing the extra space